apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/FissClaimRdaSink.java

```diff
@@ -58,6 +58,10 @@ RdaClaimMessageMetaData createMetaData(RdaChange<RdaFissClaim> change) {
         .receivedDate(claim.getLastUpdated())
         .locations(new StringList().add(claim.getCurrLoc1()).addIfNonEmpty(claim.getCurrLoc2()))
         .transactionDate(claim.getCurrTranDate())
+        .phase(change.getSource().getPhase())
+        .phaseSeqNum(change.getSource().getPhaseSeqNum())
+        .extractDate(change.getSource().getExtractDate())
+        .transmissionTimestamp(change.getSource().getTransmissionTimestamp())
```

Comment:
This will explode if `change.getSource()` returns null.  Is that not possible?

Reply:
Fiss and MCS transformers call transformSource() (https://github.com/CMSgov/beneficiary-fhir-data/blob/kylerobbins/paca-690-rda-version-0.8/apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/FissClaimTransformer.java#L496-L519) which always returns at least an empty object (https://github.com/CMSgov/beneficiary-fhir-data/blob/kylerobbins/paca-690-rda-version-0.8/apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/AbstractClaimTransformer.java#L25-L59)